examples/remote-offline-store/README.md

```diff
@@ -0,0 +1,140 @@
+The way we implement the parameter transfer protocol is the following:
+* The client, e.g. the `RemoteOfflineStore` instance, receives a call to the `get_historical_features` API with the required parameters 
+(e.g., `entity_df` and `feature_refs`).
+* The client creates a unique identifier for a new command, using the UUID format, and generates a `Flight Descriptor` to represent it.
+* Then the client sends the received API parameters to the server using multiple calls to the `do_put` service
+  * Each call includes the data stream of the parameter value (e.g. the `entity_df` DataFrame).
+  * Each call also adds additional metadata values to the data schema to let the server identify the exact API to invoke:
+      * A `command` metadata with the unique command identifier calculated before.
+      * An `api` metadata with the name of the API to be invoked remotely, e.g. `get_historical_features`.
+      * A `param` metadata with the name of each parameter of the API.
+* When the server receives the `do_put` calls, it stores the data in memory, using an ad-hoc `flights` dictionary indexed by the unique 
+`command` identifier and storing a document with the streamed metadata values:
+```json
+{
+    "(b'8d6a366a-c8d3-4f96-b085-f9f686111815')": {
+        "command": "8d6a366a-c8d3-4f96-b085-f9f686111815",
+        "api": "get_historical_features",
+        "entity_df": ".....",
+        "features": "...."
+    }
+}
+```
+* Indexing a flight descriptor by a unique `command` identifier enables the server to efficiently handle concurrent requests from multiple clients
+for the same service without any overlaps.
+* Since the client implementation is lazy, the returned instance of `RemoteRetrievalJob` invokes the `do_get` service on the server only when
+the data is requested, e.g. in the `_to_arrow_internal` method.
+* When the server receives the `do_get` request, it unpacks the API parameters from the `flights` dictionary and, if the requested API is
+set to `get_historical_features`, forwards the request to the internal instance of `FeatureStore`.
+* Once the `do_get` request is consumed, the associated flight is removed from the `flights` dictionary, as we do not expect the same 
+API request to be executed twice from any client.
+
+Other APIs of the `OfflineStore` interface can be implemented the same way, assuming that both the client and the server implementation 
+agree on the parameter transfer protocol to be used to let the server execute the service remotely.
+
+As an alternative, APIs that do not have any returned data may be implemented as a `do_action` service in the server.
+
+## Validating the POC
+### Launch the offline server
+A default Feast store has been defined in [offline_server/feature_repo](./offline_server/feature_repo/) using the `feast init` command
+and must be first initialized with:
+```console
+cd offline_server
+feast -c feature_repo apply
```

Comment:
is this need to be run under example directory ? like
`cd examples/remote-offline-store/offline_server/`

Reply:
from the same folder where this README is located